pkg/printers/internalversion/describe.go

```diff
@@ -3369,6 +3369,56 @@ func (s *StorageClassDescriber) Describe(namespace, name string, describerSettin
 	return describeStorageClass(sc, events)
 }
 
+func printAllowedTopology(w PrefixWriter, topology *storage.VolumeProvisioningTopologies) {
+	w.Write(LEVEL_0, "Allowed Topologies:\t")
+	if topology == nil || len(topology.RequiredTopologies) == 0 {
+		w.WriteLine("<none>")
+		return
+	}
+	w.WriteLine("")
+
+	w.Write(LEVEL_1, "RequiredTopologies:\n")
+	printRequiredTopologiesMultilineWithIndent(w, LEVEL_2, topology.RequiredTopologies)
+}
+
+// printRequiredTopologiesMultilineWithIndent prints multiple topologies with a user-defined alignment.
+func printRequiredTopologiesMultilineWithIndent(w PrefixWriter, indentLevel int, topologies []storage.VolumeTopology) {
+	for i, topology := range topologies {
+		if i > 0 {
+			w.Write(indentLevel, "OR\n")
+		}
+		// to print labels in the sorted order
+		keys := make([]string, 0, len(topology.MatchNodeLabels))
+		for key, labels := range topology.MatchNodeLabels {
+			keys = append(keys, fmt.Sprintf("[%s=%s]", key, labels))
+		}
+		sort.Strings(keys)
+
+		exprStr := strings.Join(keys, " AND ")
+		w.Write(indentLevel, "%s\n", exprStr)
+	}
+}
+
+func printCapacityInStorageClass(w PrefixWriter, capacity *storage.StorageClassCapacity) {
+	w.Write(LEVEL_0, "Capacity:\t")
```

Comment:
Is this the best practice? `Capacity: <none>` even when capacity is nil? (asking since this is just a alpha feature, it should not even show up if it is not set right?)